src/pages/Task.tsx

```diff
@@ -56,14 +56,13 @@ export const Task = () => {
     displayName,
     patchNumber,
     priority,
-    status,
     annotation,
     latestExecution,
     versionMetadata,
     displayTask,
   } = task ?? {};
   const attributed = annotation?.issues?.length > 0;
-
+  const status = null;
   if (
```

Comment:
I think passing `null` to the `TaskStatusBadge` results in an empty badge always, even if the task did have a proper status. The task shown below was successful, so ideally it should have the `SUCCEEDED` badge:

<img width="292" alt="Screen Shot 2022-05-25 at 3 03 37 PM" src="https://user-images.githubusercontent.com/47064971/170348428-b8ba71e2-f18c-44bd-bce4-e6f95e7966e1.png">

Maybe instead we can edit `TaskStatusBadge` to return `null` if the status is `null` (or you could make it say something like "Unknown", whatever you think works best), otherwise return the normal badge?